llvm/CMakeLists.txt

```diff
@@ -256,7 +256,12 @@ unset(SHOULD_ENABLE_PROJECT)
 # Build llvm with ccache if the package is present
 set(LLVM_CCACHE_BUILD OFF CACHE BOOL "Set to ON for a ccache enabled build")
 if(LLVM_CCACHE_BUILD)
-  find_program(CCACHE_PROGRAM ccache)
+  if(${CMAKE_SYSTEM_NAME} MATCHES "AIX")
```

Comment:
This tool is actually available on other platforms too:
https://packages.ubuntu.com/search?searchon=contents&keywords=ccache-swig&mode=exactfilename&suite=kinetic&arch=any

I think we should just use `find_program` to see if `ccache` exists, and if that returns `NOTFOUND` the fall back to looking for  `ccache-swig` with find program
https://cmake.org/cmake/help/latest/command/find_program.html

Reply:
Ah thanks! Fixed!